rack-protection/rack-protection.gemspec

```diff
@@ -18,6 +18,21 @@ Gem::Specification.new do |s|
     "rack-protection.gemspec"
   ]
 
+  if s.respond_to?(:metadata)
+    s.metadata = {
+      'source_code_uri'   => 'https://github.com/sinatra/sinatra/tree/master/rack-protection',
+      'homepage_uri'      => 'http://sinatrarb.com/protection/',
+      'documentation_uri' => 'https://www.rubydoc.info/gems/rack-protection'
+    }
+  else
+    msg = "RubyGems 2.0 or newer is required to protect against public "\
+          "gem pushes. You can update your rubygems version by running:\n\n"\
+          "gem install rubygems-update\n"\
+          "update_rubygems\n"\
+          "gem update --system"
+    raise msg
```

Comment:
No need to assign a local variable, I think.
Instead, you can use here-document.
Please take a look at the following code:

```ruby
  raise <<-EOF
RubyGems 2.0 or newer is required to protect against public
gem pushes. You can update your rubygems version by running:
  gem install rubygems-update
update_rubygems:
  gem update --system
  EOF
```